api/graphql/schema.graphql

```diff
@@ -1620,6 +1636,10 @@ type Mutation {
     @delete
     @guard
     @can(ability: "delete", find: "id")
+  assignUsersToTeam(
```

Comment:
I'm not sure I love this name.  This mutation can be used to remove users from teams, too, right?  Maybe `updateUserTeamAssignments`?